src/dummy_daos/daos.h

```diff
@@ -16,12 +16,12 @@
 
 #pragma once
 
-#include <stdlib.h>
-#include <stdint.h>
+#include <cstdlib>
+#include <cstdint>
 #include <unistd.h>
 #include <uuid/uuid.h>
-#include <stdbool.h>
-#include <string.h>
+#include <cstdbool>
```

Comment:
This one is deprecated with C++17 and should be removed as c++ has true/false as keywords